src/utils/util.js

```diff
@@ -82,6 +82,22 @@ function isPluginUrl(serverUrl, inputURL) {
   return server.origin === parsedURL.origin && parsedURL.pathname.toLowerCase().startsWith(`${server.subpath}plugins/`);
 }
 
+function isPluginOAuth(inputURL) {
```

Comment:
Hey @larkox, thanks for being so active in the community - and a release MVP as well! 🙂

I've a few recommendations on the implementation of this function (`isPluginOAuth`):

- Having `[A-Za-z]` and the `i` flag is redundant, so we could remove the flag in this case (in the suggested implementation below, I've used `\w`, which is equivalent to `[A-Za-z0-9_]`)
- Since we are only interested in knowing whether the pattern matches the `RegExp` or not (we are not using the matches from the `match` function), it is better to use `RegExp.test()`, which is faster than the `match` function. Using `test` gives the bonus of returning its result directly from `isPluginOAuth` instead of adding conditions and returning `true` or `false` explicitly
- We could use an optional `2` at the end of the `RegExp`, like `oauth[2]?` and avoid the need for the array and the for loop
- I think hyphens and underscores are also possible in the url before `/oauth/`, right?
- I don't know if the scope of this function should handle edge cases for `.` in the url, like 2 consecutive ones `..`, or a url like `/plugins/./oauth/`

I suggest an implementation like the following (not handling edge cases for `.`)

```javascript
function isPluginOAuth(inputURL) {
  const parsedURL = parseURL(inputURL);
  return /^\/plugins\/[\w-.]+\/oauth[2]?\//.test(parsedURL.pathname);
}
```
Hope this helps 😊. BYW, I'm not familiar with MM plugins yet, just the `RegEx` stuff got me a bit curious 🙃